File: clients/dotnet/enums/MatchingTaxType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum MatchingTaxType
    {
        /// <summary>
        /// 
        /// </summary>
        All,

        /// <summary>
        /// 
        /// </summary>
        BothSalesAndUseTax,

        /// <summary>
        /// 
        /// </summary>
        ConsumerUseTax,

        /// <summary>
        /// 
        /// </summary>
        MedicalExcise,

        /// <summary>
        /// 
        /// </summary>
        Fee,

        /// <summary>
        /// 
        /// </summary>
        VATInputTax,

        /// <summary>
        /// 
        /// </summary>
        VATNonrecoverableInputTax,

        /// <summary>
        /// 
        /// </summary>
        VATOutputTax,

        /// <summary>
        /// 
        /// </summary>
        Rental,

        /// <summary>
        /// 
        /// </summary>
        SalesTax,

        /// <summary>
        /// 
        /// </summary>
        UseTax,


    }
}

File: clients/php/enums/MatchingTaxType.php
<?php
/*
 * AvaTax Enum Class
 *
 * (c) 2004-2016 Avalara, Inc.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
namespace Avalara.AvaTax;

/**
 * @author Ted Spence <ted.spence@avalara.com>
 * @author Bob Maidens <bob.maidens@avalara.com>
 */class MatchingTaxType extends AvaTaxEnum 
{

    const All = "All";
    const BothSalesAndUseTax = "BothSalesAndUseTax";
    const ConsumerUseTax = "ConsumerUseTax";
    const MedicalExcise = "MedicalExcise";
    const Fee = "Fee";
    const VATInputTax = "VATInputTax";
    const VATNonrecoverableInputTax = "VATNonrecoverableInputTax";
    const VATOutputTax = "VATOutputTax";
    const Rental = "Rental";
    const SalesTax = "SalesTax";
    const UseTax = "UseTax";
}
